experimental/devices/hx711/hx711.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package hx711 implements an interface to the HX711 analog to digital converter.
+//
+// Datasheet
+//
+// https://www.mouser.com/ds/2/813/hx711_english-1022875.pdf
```

Comment:
The original source is http://www.aviaic.com/Download/hx711F_EN.pdf.pdf (yes with two .pdf) so please use this even if _iffy_.